examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -0,0 +1,434 @@
+    -->
+    <select>
+      <option value="watching">Watching</option>
+      <option value="only-mentions">Only mentions</option>
+      <option value="ignoring">Ignoring</option>
+    </select>
+
+    <!--
+      #### Step 2: Fetching the Current Subscription Setting
+
+      To query the server for the user's current subscription setting on email or page load and pre-select the user's current setting in the dropdown, we wrap the `select` element in an `amp-list` and set its `src` to the URL where our server is listening for GET requests.
+
+      We use an `amp-mustache` template to render the `select` element and ensure that the `option` element corresponding to the user's subscription setting has the `selected` attribute. To understand why the code is duplicative, consider that the `selected` attribute is a [boolean attribute](https://www.w3.org/TR/2011/WD-html5-author-20110809/common-microsyntaxes.html#boolean-attribute) and review the [`amp-mustache` restrictions](/documentation/components/amp-mustache/#restrictions).
+
+      Finally, we add a [placeholder and fallback](/documentation/guides-and-tutorials/develop/style_and_layout/placeholders) to keep the user informed about the state of the user interface and gracefully handle any issues.
+    -->
+    <amp-list src="<% base_path %>/subscription" binding="refresh" items="." single-item layout="fill">
+      <template type="amp-mustache">
+        <select>
+          {{#options}}
+            {{#isSelected}}
+              <option value="{{value}}" selected>
+                {{text}}
+              </option>
+            {{/isSelected}}
+            {{^isSelected}}
+              <option value="{{value}}">
+                {{text}}
+              </option>
+            {{/isSelected}}
+          {{/options}}
+        </select>
+      </template>
+      <div placeholder>Loading...</div>
+      <div fallback>Something went wrong. Please refresh</div>
+    </amp-list>
+
+    <!--
+      Note that the `amp-list` is annotated with [`single-item`](/documentation/components/amp-list#single-item) because our server responds to GET requests with a single logical value (the subscription setting) and we only want the template to be invoked once for the response. Setting [`items="."`](/documentation/components/amp-list#items) ensures that the template can access the response object's root fields.
+    -->
+
+    <!--
+      #### Step 3: Updating the Subscription Setting on Selection
+
+      To update the user's subscription setting on the server when they select a new setting, we start by wrapping the `select` element in an [AMP `form`](/documentation/components/amp-form) element, setting its `method` to `post`, and setting its `action-xhr` attribute to the URL where our server is listening for POST requests.
```

Comment:
```suggestion
      To update the user's subscription setting on the server when they select a new setting, we start by wrapping the `select` element in an [`amp-form`](/documentation/components/amp-form) element, setting its `method` to `post`, and setting its `action-xhr` attribute to the URL where our server is listening for POST requests.
```

Reply:
I wasn't sure about this one because the usage in the code is still `form` (not `amp-form`). Do you still think we should change it to `amp-form`?

Reply:
hmm, valid point! I think the distinction here is we're saying "element" instead of "component" - it is called an `amp-form` component, but the element is just `form`. In this case, you can leave as AMP `form`.